### PR TITLE
Add more functionality to cvdr connect and mark version 0.2.0 for unstable release

### PR DESCRIPTION
**Pull request by jemoreira**

### CODE REVIEW

File: pkg/cli/cli.go [hunk 1]
```diff
@@ -395,35 +388,35 @@ func cvdCommands(opts *subCommandOpts) []*cobra.Command {
 }
 
 func connectionCommands(opts *subCommandOpts) []*cobra.Command {
-	connectFlags := &ConnectFlags{opts.RootFlags, ""}
+	connFlags := &ConnectFlags{opts.RootFlags, "", false /*skipConfirmation*/}
```
Comment (ser-io) [inline pkg/cli/cli.go:391]: nit: let's use `field: value` declaration to initialize here, I think it still could be in one line.

**Review by ser-io — CHANGES_REQUESTED** [review]
let's move `ConnectDevice`, and `gatherDisconnectedDevices` to a different file keeping cli.go about commands layout.

**Comment by jemoreira** [comment]
> let's move ConnectDevice, and gatherDisconnectedDevices to a different file keeping cli.go about commands layout.

Done

File: pkg/cli/cli.go [hunk 2]
```diff
@@ -232,6 +227,35 @@ func PromptSelection[T any](c *command, choices []T, toStr func(T) string, selOp
 	return choices, nil
 }
 
+func PromptSelectionFromMap[K comparable, T any](c *command, choices map[K]T, toStr func(K, T) string, selOpt SelectionOption) (map[K]T, error) {
```
Comment (ser-io) [inline pkg/cli/cli.go:230]: let's add some comment about how this prompt looks like to understand when it can be used.

**Review by ser-io — APPROVED** [review]

File: pkg/cli/cli.go (hunk 2 above)
Comment (jemoreira) [reply pkg/cli/cli.go:237]: done

**Pull request merged**